index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Recipe App</title>
+    <link rel="stylesheet" href="./stylesheet.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav id="navBar">
+      <a id="logo">YummyDish</a>
+      <!-- <div class="desktopMenu">
+        <a class="menuItem">Home</a>
+        <a class="menuItem">Recipes</a>
+        <a class="menuItem">Categories</a>
+        <a class="menuItem">Blog</a>
+        <a class="menuItem">Features</a> -->
+      <!-- </div> -->
+      <h2>Recipes for Every Taste</h2>
+      <button id="addRecipeBtn" class="buttonstyle" onclick="addRecipeClick()">
```

Comment:
Add event listeners in your JS file. It is more manageable to have everything in the same place IMO.